Dockerfile

```diff
@@ -17,8 +18,17 @@ RUN     apk add --no-cache git bind-tools libevent-dev openssl-dev gnupg gcc mak
     make && make install && \
     cd .. && \
     rm -rf tor && \
+    git clone https://git.torproject.org/torsocks.git /usr/local/src/torsocks && \
+    cd /usr/local/src/torsocks && \
+    git checkout $torsocks_version && \
+    apk add --no-cache libtool && \
+    ./autogen.sh && \
+    ./configure && \
+    make && make install && \
+    cd .. && \
+    rm -rf torsocks && \
```

Comment:
I think it should be better to add another RUN instruction for torsock for a better readability, it won't be a issue for image size as long we remove every installed packages on each RUN instructions.

Reply:
Regarding image size, I plan another PR that uses a fresh image for runtime. Currently it inherits its context from the build image and the current directory of the docker invocation. The way it is currently done pulls in tons and tons of MB of useless (and potentially dangerous) binaries and data.

Reply:
Ok, I extracted my changes into a separate, self-contained RUN instruction.